tests/test_tf_converter.py

```diff
@@ -351,10 +383,14 @@ def test_convnet(self):
     # not batched
     self._test_tf_model(graph,
         {"test_convnet/input:0":[1,8,8,3]}, output_name, delta=1e-2)
+    # quantized
+    self._test_tf_model(graph,
```

Comment:
I think it would be better to just make another test called "test_convnet_quantized" which calls this function "test_convnet" , with is_quantized=True. 
That way there will be 2 separate tests testing two different things.

Reply:
Sure. Thanks.